spinnaker/spinnaker_system/google_http_lb_upsert_scenario.py

```diff
@@ -181,62 +181,63 @@ def _add_contract_clauses(self, contract_builder, upsert):
     health_checks = [service['healthCheck'] for service in backend_services]
 
     hc_clause_builder = (contract_builder
-                         .new_clause_builder('Health Checks Created',
-                                             retryable_for_secs=30)
-                         .list_resource('httpHealthChecks'))
+      .new_clause_builder('Health Checks Created', retryable_for_secs=30)
+      .list_resource('httpHealthChecks')
+      )
     for hc in health_checks:
-      hc_clause_builder.contains_pred_list(
-        [
-          jp.PathEqPredicate('name', hc['name']),
-          jp.PathEqPredicate('requestPath', hc['requestPath']),
-          jp.PathEqPredicate('port', hc['port'])
-        ]
+      hc_clause_builder.contains_match({
+          'name' : jp.STR_EQ(hc['name']),
```

Comment:
I would eventually opt for more verbose names than `STR_EQ`, etc. `STR_EQ` is a trivial case, but I can imagine users of this being confused and continually looking up what these mean.